frontend/packages/component-library/src/cms/header/header.module.scss

```diff
@@ -82,7 +83,7 @@ $box-shadow: 0 2px 6px 0 rgb(0 0 0 / 0.08);
 }
 
 // Button links styles
-.buttonLinks {
+body .buttonLinks {
```

Comment:
Needed to add these to make this work in Contenful. This prevents anything else from overriding these styles.